### PR TITLE
Feature/sprint4 api development

### PR DESCRIPTION
**Pull request by FurkanKayar55**
package reclaim implementation for all package types

#### _Checklist_
- [] Package reclaim 
- [] Seeding options

### CODE REVIEW

File: LBH.AdultSocialCare.Api/V1/Controllers/DayCarePackageReclaimController.cs [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        [ProducesResponseType(typeof(IEnumerable<HomeCarePackageReclaimCategoryResponse>), StatusCodes.Status200OK)]
+        [ProducesDefaultResponseType]
+        [HttpGet]
+        [Route("reclaim-category")]
+        public async Task<ActionResult<IEnumerable<HomeCarePackageReclaimCategoryResponse>>> GetTypeOfReclaimCategoryOptionList()
+        {
+            var result = await _getAllHomeCareReclaimCategoryUseCase.GetAllAsync().ConfigureAwait(false);
+            return Ok(result);
+        }
+    }
```
Comment (dunnudge) [inline LBH.AdultSocialCare.Api/V1/Controllers/DayCarePackageReclaimController.cs:93]: I think the naming is intentional. Can we have the common names as `PackageReclaimCategoryResponse`, `PackageReclaimFromResponse`, `PackageReclaimAmountOptionResponse` without a specific care package prefix instead. Same for the domain naming

File: LBH.AdultSocialCare.Api/V1/Controllers/NursingCareBrokerage/NursingCareBrokerageController.cs [hunk 2]
```diff
@@ -18,6 +17,56 @@ namespace LBH.AdultSocialCare.Api.V1.Controllers.NursingCareBrokerage
...
+
+        //    if (!ModelState.IsValid)
+        //    {
+        //        return UnprocessableEntity(ModelState);
+        //    }
+
+        //    var nursingCareBrokerageCreationDomain = nursingCareBrokerageCreationRequest.ToDomain();
+        //    var result = await _createNursingCareBrokerageUseCase.ExecuteAsync(nursingCarePackageId, nursingCareBrokerageCreationDomain).ConfigureAwait(false);
+        //    return Ok(result);
+        //}
```
Comment (dunnudge) [inline LBH.AdultSocialCare.Api/V1/Controllers/NursingCareBrokerage/NursingCareBrokerageController.cs:70]: ?

**Review by dunnudge — CHANGES_REQUESTED** [review]

**Comment by dunnudge** [comment]
I forgot to add. Most important is there are new entities. Please run migrations

File: LBH.AdultSocialCare.Api/V1/Controllers/DayCarePackageReclaimController.cs (hunk 1 above)
Comment (FurkanKayar55) [reply LBH.AdultSocialCare.Api/V1/Controllers/DayCarePackageReclaimController.cs]: yes definitely, I miss that.

**Pull request merged**